### DISCUSSION TITLE
Need help distinguishing b/w Max Fee and Max Priority Fee

### DISCUSSION THREAD

**Discussion by ropats16** [Cyfrin/foundry-full-course-cu#1254] [Q&A]
In the Blockchain Basics Course Lesson No 9, Max Priority Fee is referred to as the Max Fee plus the tip given to the miners. However, in the example, the Max Priority Fee is actually less than the Max Fee.

So when it is said that  Max Priority Fee is referred to as the Max Fee plus the tip given to the miners:

Does it mean that total fee = X Max Fee + Y Max Priority Fee?

Or does the Y Max Priority Fee include the X Max Fee?

**Comment by ghost** [comment]
Currently, tx fees are calculated using the [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), this eip introduced new gas fields to compute the amount to pay for transactions, `maxFeePerGas` and `maxPriorityFeePerGas`.

- `maxFeePerGas`: is the total amount of fee to pay for the tx.

- `maxPriorityFeePerGas`: is the amount of tip to pay to the miner.

Also, there is a `baseFeePerGas` which is the minimum amount of fee to pay for sending the tx and is set by the network. 

The `maxFeePerGas` it's just the sum of the other fields:

```
maxFeePerGas = maxPriorityFeePerGas + baseFeePerGas
```

So yeah, the tip to the miner cannot be greater than the max fee paid for the tx.

I'll leave this article from Alchemy with more info: https://docs.alchemy.com/docs/maxpriorityfeepergas-vs-maxfeepergas

**Reply by ropats16** [reply]
Aah gotcha, thanks for clearing that up!
